pkg/kustomize/kustomize.go

```diff
@@ -0,0 +1,202 @@
+/*
+Copyright 2019 The Kubermatic Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package kustomize
+
+import (
+	"fmt"
+	"io"
+	"net/http"
+	"os"
+
+	statikfs "github.com/thetechnick/statik/fs"
```

Comment:
btw, this should be moved to utils repo as well. It has no business being in private member repo.